### PR TITLE
Beta release notes for 0.113

### PR DESCRIPTION
**Pull request by frenck**
## Proposed change

Beta release notes for Home Assistant Core 0.113.0

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_posts/2020-07-01-release-113.markdown [hunk 1]
```diff
@@ -0,0 +1,853 @@
...
+
+<p class='img'>
+<img src='/images/blog/2020-07-0.112/datetime-picker.png' alt='Screenshot of lorem ipsum'></a>
+Screenshot of lorem ipsum.
+</p>
+
+## MDI icons updated
+
+It has taken some time for us to upgrade to the newest version of Material Design Icons, 5.3.45, there was a reason for that, version [5.0.45](https://dev.materialdesignicons.com/upgrade#4.9.95-to-5.0.45) contains a lot of breaking changes.
+We wanted to handle these well so it tooks some time.
```
Comment (iantrich) [inline source/_posts/2020-07-01-release-113.markdown]:
```suggestion
We wanted to handle these well so it took some time.
```

File: source/_posts/2020-07-01-release-113.markdown [hunk 2]
```diff
@@ -0,0 +1,853 @@
...
+
+A lot of icons are renamed, and some were removed.
+In this release we included all new, and all removed icons. Also will the new and the old name work.
```
Comment (iantrich) [inline source/_posts/2020-07-01-release-113.markdown]: `Also will the new and the old name work.`
I don't understand what this section means

File: source/_posts/2020-07-01-release-113.markdown [hunk 3]
```diff
@@ -0,0 +1,853 @@
...
+
+If you use an icon that is renamed of removed we will show a warning in the log, in version 0.115 we will remove this conversion path and will removed icons and old names no longer work.
```
Comment (iantrich) [inline source/_posts/2020-07-01-release-113.markdown]:
```suggestion
If you use an icon that is renamed of removed we will show a warning in the log, in version 0.115 we will remove this conversion path and removed icons and old names no longer work.
```

File: source/_posts/2020-07-01-release-113.markdown [hunk 4]
```diff
@@ -0,0 +1,853 @@
...
+
+The UI to edit or create a script has been updated, you can now define your own ID for a script, so you no longer have to search your states for a long numeric entity id for every script.
+We also added support for the new `mode` and you can give your scripts a custom icon from the UI.
+
+The icon was also added to the scenes editor.
+
+## Deprecated HTML imports
+
+With the start of custom components, you would import a HTML file for your component instead of JavaScript. That's why we have always supported importing extra HTML in the frontend and custom panels.
+This has been deprecated and superseded by ES modules since some time and has (almost) no support anymore in browsers. We have a polyfill in place to still support this, but we are going to remove this.
```
Comment (iantrich) [inline source/_posts/2020-07-01-release-113.markdown]:
```suggestion
This has been deprecated and has (almost) no support anymore in browsers. They have been superseded by ES modules for awhile, but we have a polyfill in place to still support this, but we are going to remove this in the near future.
```

**Review by balloob — APPROVED** [review]

**Pull request merged**